h2o-py/tests/testdir_generic_model/pyunit_generic_model_mojo_cox_ph.py

```diff
@@ -26,17 +26,25 @@ def test(output_test, x, ties, stratify_by, use_all_factor_levels):
         ties=ties
     )
     coxph.train(x=x, y="event", training_frame=heart)
-    with Capturing() as original_output:
-        coxph.show()
-        
     coxph.show()
 
-    original_model_filename = tempfile.mkdtemp()
-    original_model_filename = coxph.download_mojo(original_model_filename)
-      
-    model = H2OGenericEstimator.from_file(original_model_filename)
+    mojo_path = tempfile.mkdtemp()
```

Comment:
it's usually recommended to clean up tmp folders automatically:
```python
with pyunit_utils.TemporaryDirectory() as tmpd:
    mojo_path = coxph.download_mojo(tmpd)

    from h2o.estimators import H2OGenericEstimator
    …
```

Reply:
@sebhrusen thanks for the suggestion, you are right

I tried your solution but it is not available on the release branch - I used the "results" directory where we typically store files like that